src/algorithms.ts

```diff
@@ -50,3 +51,38 @@ export function intersectTokenScores(arrays: TokenScore[][]): TokenScore[] {
 
   return result;
 }
+
+export function prioritizeTokenScores(arrays: TokenScore[][]): TokenScore[] {
+  const tokenMap: Record<string, number> = {};
+
+  const mapsLength = arrays.length;
+  for (let i = 0; i < mapsLength; i++) {
+    const arr = arrays[i];
+
+    const entriesLength = arr.length;
+    for (let j = 0; j < entriesLength; j++) {
+      const [token, score] = arr[j];
+
+      if (token in tokenMap) {
+        tokenMap[token] += score + 0.5
```

Comment:
Should we make this `0.5` weight parametrizable?

Reply:
Yes. This way we give people chances to spot how the score was manipulated.

Reply:
alright